src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -108,21 +108,25 @@ class TezosNodeOperator(node: TezosRPCInterface) extends LazyLogging {
     * @param hash     Hash of the given block
     * @return         Block
     */
-  def getBlock(network: String, hash: String): Try[TezosTypes.Block] =
-    node.runGetQuery(network, s"blocks/$hash").flatMap { jsonEncodedBlock =>
+  def getBlock(network: String, hash: String, offset: Option[Int] = None): Try[TezosTypes.Block] = {
+    val offsetString = offset.map(_.toString).getOrElse("")
```

Comment:
maybe can be shortened?

I'll try something like this, if it's not too compact to the point of being cryptic

```scala
val offsetString = offset.fold("")(off => s"~$off")
```

Reply:
i.e. `option.fold(a)(f)`is exactly the same as `option.map(f).getOrElse(a)`